Added Epson ET-15000

rmalbrecht opened the pull request:
Added Epson ET-15000

<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->
## Device information
<!--
  Provide and links and additional information about the device you are adding in this PR.
-->

## Home Assistant Device information
<!--
  Provide the Home Assistant device information. This can be found on the device page in HA, on the top left under `Device Info`.
  Please paste that information here.
-->

## Checklist
<!--
  Please check all the boxes when applicable.
-->

- [ ] I have created a single PR per device. When you have multiple submissions please create separate PR's.
- [ ] For lights - I have only included the gzipped files (*.gz), not the raw CSV files.
- [ ] For lights - I have provided a CSV file per supported color mode. Look that up in Developer Tools -> States

## Additional info
<!--
  Add any additional info we must know about the measurements here.
-->

bramstroker commented:
Could you share the device info from left top of device page?

<img width="356" height="162" alt="Screenshot 2025-12-14 at 12 50 23" src="https://github.com/user-attachments/assets/b45e48bd-b8c2-4f60-b830-74d6f2d049bc" />

bramstroker commented on profile_library/epson/epson/ET-15000 Series/model.json:
```diff
@@ -0,0 +1,17 @@
+  "fixed_config": {
+    "states_power": {
+      "stopped": 0.54,
+      "idle": 7,
```
What's the difference between these two states?

bramstroker replied:
> 1. Switched off (standby)
> 2. switched on but not printing
> 3. switched on and printing
> 

Looking at the screenshot, currently the epson is unavailable.
Did you actually disconnected it or is this some sort of sleep mode where wifi is turned off making the printer unavailable to HA.

rmalbrecht commented:
1. switched on & printing 14 watt
2. switched on & not printing: idle 7 watt
3. after  60 minutes of idle  ->  standby: 0,5 watt
5. after 4 hours of standby -> switch off -> unavailable, nothing that I can measure

What Epson says:
12 W (Kopieren ohne PC, ISO/IEC-Muster 24712), 1 W (Energiesparmodus), 5,4 W (Betriebsbereit), 0,2 W (ausgeschaltet), TEC 0,3 kWh/week

Epsons values are (unsurprisingly) a bit lower 
0,2 Watt for off. That is nothing my devices can reliable measure.

bramstroker commented:
> * switched on & not printing: idle 7 watt

This is `idle` state in HA?

> * after  60 minutes of idle  ->  standby: 0,5 watt

This is `stopped` state in HA right?
So it actually goes from `idle` -> `stopped` after 60 minutes, is that correct?

> * after 4 hours of standby -> switch off -> unavailable, nothing that I can measure

We can use 0.2W for that. 
Maybe we can utilize [sleep power](https://docs.powercalc.nl/configuration/standby-power/?h=sleep#sleep-power) option for that.
But I must test that if it's working correctly when the printer entity gets unavailable.

Could you provide some state history, maybe the state history graph for one day, where also a printing job was done.
Maybe you can put the power sensor chart under it, when you still have the smart plug connected.
That will definitely help to see what state changes happens, and I will add a unit tests to simulate that behaviour, and to verify the profile we build will behave correctly.

rmalbrecht commented:
Yes, this is also my understanding.
The IPP integration is mainly intended to monitor the ink levels; they don't change that fast. It seems to poll the printer state once an hour. Catching certain printer states needs some luck and tweaking:

![Bildschirmfoto 2025-12-15 um 08 04 56](https://github.com/user-attachments/assets/3decd0a1-9aad-4800-8cc6-38b50d23d9f6)

Also, the IPP integration state entity is disabled by default. Unless the user manually enables it, it's not there.

![Bildschirmfoto 2025-12-15 um 08 08 28](https://github.com/user-attachments/assets/7d0160e0-f5b3-44ce-9f18-a464a3ef4376)

In the usual household the printer will stay switched off 99% of the time and switched on when some printing is needed. Then it goes to idle after some (from the vendor) pre-configured and then finally switch off. The manual says, that this behavior also differs between countries.
